scripts/vagrant/README.md

```diff
@@ -0,0 +1,68 @@
+Vagrant setup
+===
+
+## Prerequisites
+
+- [Create a Github app](https://github.com/settings/apps) and download the private key from the app's page.
+- Put it under `$PROJECT_ROOT/.secrets/builder-github-app.pem`
+- Create a new file at `$PROJECT_ROOT/.secrets/habitat-env` and write the following into it:
+
+```
+export APP_HOSTNAME=localhost:3000
+
+export GITHUB_API_URL=https://api.github.com
+export GITHUB_WEB_URL=https://github.com
+
+export GITHUB_APP_ID=""
+
+export GITHUB_CLIENT_ID=""
+export GITHUB_CLIENT_SECRET=""
+
+export WORKER_AUTH_TOKEN=""
+
+export GITHUB_ADMIN_TEAM=""
+export GITHUB_WORKER_TEAM=""
+
+export GITHUB_WEBHOOK_SECRET=""
+```
+
+- Ask a cowrecker for secrets (easy). Or generate your own (painful).
```

Comment:
s/crowrecker/coworker 😆

Reply:
That's intentional. Because we "wreck" things together. 😛